src/midi/midi_fluidsynth.cpp

```diff
@@ -175,7 +175,9 @@ static std::string find_sf_file(const std::string &name)
 
 MidiHandlerFluidsynth::MidiHandlerFluidsynth()
         : soft_limiter("FSYNTH", prescale_level, FRAMES_PER_BUFFER)
-{}
+{
+	keep_rendering = false;
```

Comment:
Atomic types normal don't allow an initial value, however I believe the atomic book is unique in this regard, so I might be able to move this into the constructor's initialization block.  Will try.

Reply:
Just note: atomic values are not necessarily faster than values hidden behind mutexes - they are just atomic.

Reply:
> Just note: atomic values are not necessarily faster than values hidden behind mutexes - they are just atomic.

Indeed; they are sure nicer to work with though (in the form of a single variable, with mutual-access guaranteed behind the scene), versus carrying a mutex around and needing several lines of code to read or write values.

Reply:
Also good news - the atomic_bool can be initialized in the constructor.  Update inbound.